### ISSUE TITLE
Bad judgment :isValid

### ISSUE THREAD

**Issue by KilianShen** [iamkun/dayjs#2782]
**Describe the bug**

**Expected behavior**

**Information**
 - Day.js Version [e.g. v1.11.13]
 - OS: [e.g. win]
 - Browser [e.g. chrome 113]
 - Time zone: [e.g. GMT-07:00 DST (Pacific Daylight Time)]
![image](https://github.com/user-attachments/assets/25996caa-ebf3-41cf-afc2-ed180efe3760)

**Comment by RyanV-Souza** [comment]
I don't think this is a bug, because when we start a Date Object with a number, it is read as if it were seconds. For example: JavaScript is based on Epoch Time, so new Date(0) is like 1970-01-01 00:00:00 and new Date(1000) is like 1970-01-01 00:00:01.

**Comment by pccai** [comment]
![image](https://github.com/user-attachments/assets/f6e25867-ec44-4ee4-afc3-e6e3dd4db0c7)

**Comment by pccai** [comment]
`String.prototype.isValid = function() {
  return this.length > 0; //a demo
};`
